Gemfile

```diff
@@ -89,7 +89,7 @@ gem 'sidekiq-unique-jobs', '~> 7.1'
 gem 'sidekiq-bulk', '~> 0.2.0'
 gem 'simple-navigation', '~> 4.4'
 gem 'simple_form', '~> 5.2'
-gem 'stoplight', '~> 3.0.1'
+gem 'stoplight'
```

Comment:
Not sure why dropping the version specifier, since we're specifying it for every other gem.

Reply:
Rebased, and pushed commit to restore version